Updates after copy editing (#85)
mraible/jhipster-book: src/docs/asciidoc/modules/ROOT/pages/preface.adoc

I used AngularJS for several years as well. In early 2013, I first used AngularJS on a project where I implemented 40% of the code that jQuery would've required. I helped that company modernize its UI in a project for which we integrated Bootstrap. I was very impressed with both AngularJS and Bootstrap and have used them ever since. In 2014, I used Ionic on a project to implement an HTML5 UI in a native iOS application. We used AngularJS, Bootstrap, and Spring Boot in that project, which worked very well for us.

When I heard about JHipster, I was motivated to use it immediately. It combined my most-often-used frameworks into an easy-to-use package. For the first several months I knew about JHipster, I used it as a learning tool—generating projects and digging into files to see how it coded features. The JHipster project is a goldmine of information and lessons from several years of developer experience.

> I wanted to write this book because I knew all the tools in JHipster well. I wanted to further my knowledge of this wonderful project and show Java developers that they could be hip again by leveraging AngularJS and Bootstrap. JavaScript web development isn't scary—it's just another powerful platform that can improve your web-development skills.

Author: ```suggestion
I wanted to write this book because I knew all the tools in JHipster well. I wanted to further my knowledge of this wonderful project and show Java developers that they could be hip again by leveraging Angular and Bootstrap. JavaScript web development isn't scary—it's just another powerful platform that can improve your web-development skills.
```

== What is in an InfoQ mini-book?

InfoQ mini-books are designed to be concise and serve technical architects looking to get a firm conceptual understanding of a new technology or technique in a quick yet in-depth fashion. You can think of these books as covering a topic strategically or essentially. After reading a mini-book, you should have a fundamental understanding of the technology, including when and where to apply it, how it relates to other technologies, and an assimilated knowledge of other professionals who know what this technology is about. You will then be able to make intelligent decisions about the technology once your projects require it and can delve into more detailed sources (such as larger books or tutorials) at that time.

== Who is this book for?

This book is aimed specifically at web developers who want a rapid introduction to Angular, Bootstrap, and Spring Boot by learning JHipster. JHipster 5 adds support for React as well, but I won’t be covering it in this book since Angular is the default.

== What you need for this book

> To try code samples, you will need a computer running an up-to-date operating system (Windows, Linux, or macOS). You will need Node.js and Java installed. The book code was tested against Node.js 16 and Java 11.

Author: ```suggestion
To try code samples, you will need a computer running an up-to-date operating system (Windows, Linux, or macOS). You will need Node.js and Java installed. The book's code was tested using Node.js 16 and Java 11.
```

TIP: Tips are shown using callouts like this.

> WARNING: Warnings are shown using callouts like this:

Author: ```suggestion
WARNING: Warnings are shown using callouts like this.
```
